controllers/disruption_controller.go

```diff
@@ -86,7 +88,7 @@ type CtxTuple struct {
 //+kubebuilder:rbac:groups=core,resources=nodes,verbs=list;watch
 //+kubebuilder:rbac:groups=core,resources=services,verbs=list;watch
 
-func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ctrl.Result, err error) {
+func (r *DisruptionReconciler) Reconcile(_ context.Context, req ctrl.Request) (res ctrl.Result, err error) {
```

Comment:
Usually it's worrying to me to have a context not used.
It's a code smell we are not handling things correctly IMO.

We should create a dedicated task to ask ourselves in each Kubernetes API call, should it be cancelled or not (during a deploy as an example)

And add required management.

Not in this PR, but to keep in mind...

Reply:
Agreed, I do think we should start using this `ctx`, but that's a separate PR